### DISCUSSION TITLE
美國擬限制中國AI量子運算

### DISCUSSION THREAD

**Discussion by ruthchen1020** [1101-GS4719-C/Python-Community-GS4719-C#824] [General]
中國的ai一直以來走在世界尖端，因為他們擁有非常好搜集的龐大資料庫，也擁有很先進的技術。
為了防止中國用ai科技研究量子運算，美國正在擬定新的限制。
https://udn.com/news/story/6811/6705363?utm_source=lineweb&utm_medium=share

**Comment by bonny123865** [comment]
美國的行動牽一髮而動全身，直接影響了各國的生產鏈
感覺接收到這個消息之後，也應該開始制定政策應對了!

**Comment by Pietra1226** [comment]
聽說中國的超級電腦擁有目前全球最強的運算，一方面是他們蒐集資料不需要經過太多繁瑣步驟，對於AI擁有大數據就先贏一半。

**Comment by ruthchen1020** [comment]
中國AI真的太強了，畢竟全國人民的資料都是無痛蒐集。
不像美國，一直為資料蒐集打官司。

**Comment by StevenWanglolz** [comment]
美國真的很擔心其他國家威脅到他們的地位ㄟ，各種手段都做得出來~
